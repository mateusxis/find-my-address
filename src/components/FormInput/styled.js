import styled from 'styled-components';

export const SForm = styled.form`
  align-items: center;
  display: flex;
  flex-direction: column;
  padding: 1rem;
`;

export const SSection = styled.section`
  align-items: center;
  display: flex;
  margin: 30px 0;
  flex-direction: column;

  @media (min-width: 601px) {
    margin: 40px 0;
    flex-direction: row;
  }

  @media (min-width: 961px) {
    margin: 50px 0;
    flex-direction: row;
  }
`;
