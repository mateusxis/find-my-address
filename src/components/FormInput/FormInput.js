import { SForm, SSection } from './styled';
import Button from '../../container/ButtonContainer';
import Input from '../../container/InputContainer';
import Table from '../../container/TableContainer';

const FormInput = () => {
  return (
    <SForm>
      <SSection>
        <Input placeholder="Digite o cep desejado" type="text" />
        <Button>Buscar</Button>
      </SSection>
      <Table />
    </SForm>
  );
};

export default FormInput;
