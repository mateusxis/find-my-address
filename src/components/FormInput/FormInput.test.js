import renderer from 'react-test-renderer';

import FormInput from './FormInput';

jest.mock('../../container/ButtonContainer', () => jest.fn(() => <></>));
jest.mock('../../container/InputContainer', () => jest.fn(() => <></>));
jest.mock('../../container/TableContainer', () => jest.fn(() => <></>));

describe('FormInput', () => {
  it('should render FormInput', () => {
    const tree = renderer.create(<FormInput />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
