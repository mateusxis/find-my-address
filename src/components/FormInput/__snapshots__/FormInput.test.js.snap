// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormInput should render FormInput 1`] = `
<form
  className="sc-bdvvtL gMTBCA"
>
  <section
    className="sc-gsDKAQ cgvzQa"
  />
</form>
`;
